common/src/main/java/com/wynntils/core/discord/DiscordManager.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.discord;
+
+import com.wynntils.antiope.core.DiscordGameSDKCore;
+import com.wynntils.antiope.core.type.CreateParams;
+import com.wynntils.antiope.manager.activity.type.Activity;
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.components.Manager;
+import com.wynntils.core.text.PartStyle;
+import com.wynntils.core.text.StyledText;
+import com.wynntils.mc.event.TickEvent;
+import com.wynntils.models.character.type.ClassType;
+import com.wynntils.utils.mc.McUtils;
+import java.time.Instant;
+import java.util.List;
+import java.util.Locale;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class DiscordManager extends Manager {
+    private static final long DISCORD_APPLICATION_ID = 387266678607577088L;
```

Comment:
What is this magic number? Is it like a API key, something we have registered with Discord?

Reply:
In that case, I think it is a bit disturbing to hard-code it in the source. I wonder if there is a way that we can read it from a suitable json file instead. urls.json springs to mind, but that only contains URLs. Hm. I need to think a bit about this.

Reply:
This Client ID is fine to hard code, as long as it is our wynntils application that we have already setup